.github/workflows/build/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM hyperledger/indy-core-baseci:0.0.3-master
```

Comment:
Just interesting: didn't we plan to get rid of base images?

Reply:
We're using the existing base images for now.  We can re-evaluated once things are running smoothly and we've decommissioned the Jenkins pipelines.

As for the libindy version, I was going off the related Jenkins build error from this change; https://github.com/hyperledger/indy-node/pull/1645#discussion_r571456440

Reply:
Ok, thanks for clarifications!